spec/requests/notifications_spec.rb

```diff
@@ -26,8 +26,28 @@
 
     end
 
-  end
+    describe 'delete notification' do
+
+      it 'a notification is active by default' do
```

Comment:
@markmiranda can you remove this it block? in general I only test responses in request specs. this is something you could test on a model level, but it is a bit overkill.